src/generator/backend/fold.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+
+var exports = module.exports = {};
+
+const fs = require('fs-extra');
+const moment = require('moment');
+const handlebars = require('handlebars');
+const async = require('async');
+const archiver = require('archiver');
+
+
+function speakerNameWithOrg(speaker) {
+  return speaker.organisation ?
+    `${speaker.name} (${speaker.organisation})` :
+    speaker.name;
+}
+
+
+function byProperty(key) {
+  return (a, b) => {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier name 'a' is too short. (< 3)](https://www.codacy.com/app/triplez/open-event-webapp/file/2737479688/issues/source?bid=3233857&fileBranchId=3352811#l21)